<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositPageClass</class>
 <widget class="QWidget" name="DepositPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>存单管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4" colspan="8">
    <widget class="MyTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="buttonAdd">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_add.png</normaloff>:/Images/edit_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QPushButton" name="buttonDel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_remove.png</normaloff>:/Images/edit_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QPushButton" name="buttonEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/Notepad2.png</normaloff>:/Images/Notepad2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="8" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>223</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>存单共</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBoxCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> 笔</string>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>    本金</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBoxAmount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> 元</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>    利息</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QSpinBox" name="spinBoxInterests">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> 元</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>    总计</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QSpinBox" name="spinBoxTotal">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> 元</string>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header>MyTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections/>
</ui>
